test-integration.sh

```diff
@@ -1,15 +1,17 @@
 #!/bin/bash
-. "$(dirname $0)/variables.sh"
-
 set -e
 
+source "$(dirname $0)/variables.sh"
+
 TAGS="integration"
 DIR="integration"
 INTEGRATION_TIMEOUT=${INTEGRATION_TIMEOUT:-10m}
 COVERFILE=${COVERFILE:-cover.out}
+EXCLUDE_FILE=${EXCLUDE_FILE}
```

Comment:
nit: isn't this a noop?

Reply:
ended up refactoring for another reason.